src/components/TModal.vue

```diff
@@ -186,14 +186,14 @@ export default {
       };
     },
     normalizedWidth() {
-      if (! Number.isNaN(this.width)) {
+      if (Number.isNaN(this.width)) {
```

Comment:
If I pass 100 as a number or also as a string in prop

condition Number.isNaN(100) or Number.isNaN("100") returns false

so computed prop returns just 100.

The best solution for me is that, if the prop is string just return it and if it is a number, append 'px' and return

Reply:
@figurluk Thank you for your comment  😁  

I have updated by convert to `isNaN` as global, instead of `Number.isNaN`.
Because global `isNaN` will parse the value to number before tested.
So, `100` and `"100"` will be true in condition.

Reply:
I have updated.